### ISSUE TITLE
[Feature Request] Auto Scrolling for Webtoons

### ISSUE THREAD

**Issue by NaeemBolchhi** [Jays2Kings/tachiyomiJ2K#1296]
### Describe your suggested feature

It'd be nice to have automatic scrolling in webtoon view that can be turned on. It could stop scrolling when it detects a tap or manual scrolling. The scrolling speed could be customized as well.

Could this work?

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an request for a feature in the original Tachiyomi, I am opening because multiple versions of TachiyomiJ2K have been released without this feature.
- [X] If this is an issue with an extension, or a request for an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[1.5.4](https://github.com/Jays2Kings/tachiyomiJ2K/releases/latest)**.
- [X] I have checked through the app settings for my feature.
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements.
- [X] I will fill out all of the requested information in this form.

**Comment by Jjjggh** [comment]
It would be nice , 
It's already in tachiyomi sy and az

**Comment by Gamefire238** [comment]
> It would be nice , It's already in tachiyomi sy and az

indeed, but it's just like an auto click that clicks on the screen and shifts it with a delay, what the above person wants is probably the auto scroll feature like in mangatoon

**Comment by fifth-island** [comment]
IT has been three years and they haven't implemented it yet... it would be so useful!
